app/uk/gov/hmrc/gform/sharedmodel/formtemplate/HasExpr.scala

```diff
@@ -17,19 +17,25 @@
 package uk.gov.hmrc.gform.sharedmodel.formtemplate
 
 sealed trait ExprCardinality
-case class SingleExpr(expr: Expr) extends ExprCardinality
+case class SingleExpr(expr: Expr, roundingMode: Option[RoundingMode]) extends ExprCardinality
```

Comment:
Putting `roundingMode` into this class is mixing unrelated concerns. We should move extraction of `roundingMode` from `ComponentType` directly when it is needed ie.  into `Javascript.scala`.

Reply:
roundingMode has been moved to Javascript.scala